.ci/wheel/build_wheel.sh

```diff
@@ -133,6 +133,25 @@ EXTRA_CONDA_INSTALL_FLAGS=""
 CONDA_ENV_CREATE_FLAGS=""
 RENAME_WHEEL=true
 case $desired_python in
+    3.14t)
```

Comment:
Did we add the right supported classifiers in pyproject.toml? We shouldn't forget to do that.

Reply:
Hi @Skylion007 yes I believe this was already done